go-controller/pkg/metrics/ovs.go

```diff
@@ -219,6 +219,21 @@ var metricOvsInterfaceCollisionsTotal = prometheus.NewGauge(prometheus.GaugeOpts
 	Help:      "The total number of packet collisions transmitted by Open vSwitch interface(s).",
 })
 
+var metricOvsInterfaceTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+	Namespace: MetricOvsNamespace,
+	Subsystem: MetricOvsSubsystemVswitchd,
+	Name:      "interfaces_total",
+	Help:      "The total number of Open vSwitch interface(s) created for pods",
+})
+
+var MetricOvsInterfaceUpWait = prometheus.NewCounter(prometheus.CounterOpts{
+	Namespace: MetricOvsNamespace,
+	Subsystem: MetricOvsSubsystemVswitchd,
+	Name:      "interface_up_wait_total",
+	Help: "The total number of checks, every 200 milliseconds that is required to wait for pod " +
```

Comment:
If the sleep amount changes then we will probably won't notice that we need to change the metric as well.

Is it possible to measure directly the amount of time in milliseconds, so increment by 200 which could be the same variable used in the sleep?

Reply:
yes, thats a better idea that what is here currently. Its also intuitive for the user to understand versus a count of a time you need to know about.

@pperiyasamy we can do this but we must report the time in seconds not miliseconds and also the metric name to
```
interface_up_wait_seconds_total
```

Reply:
https://prometheus.io/docs/practices/naming/#base-units

Reply:
https://prometheus.io/docs/practices/naming/#metric-names

Reply:
Thanks @jcaamano and @martinkennelly. Nice thought! changed it.